content/overview/getting-started/_index.en.md

```diff
@@ -99,16 +102,17 @@ Host Inventory (NCPGH5CVPO3BAZ5OSQKXYHDKPBT3JXLG5EAOTG7XOXUWJ6AHZCFT57SI)
   MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5    N/A                wasmcloud.azurecr.io/echo:0.3.2
 
   Provider ID                                                 Name               Link Name          Image Reference
-  VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M    N/A                default            wasmcloud.azurecr.io/httpserver:0.14.4
+  VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M    N/A                default            wasmcloud.azurecr.io/httpserver:0.14.6
 ```
 
 #### Linking actors and capability providers
 
-The Echo actor and HTTP Server providers are running, but they aren't connected. Since the HTTP server provider hasn't been _linked_ to any actor yet, it hasn't yet open a port to listen to web requests to forward to the actor. To allow the actor and provider to communicate, they need to be linked. We could link them with wash cli (`wash ctl link put ...`), using the ActorID and the ProviderID from the inventory, but this time we'll link them in the web UI.
-
-Go back to the web UI and click **Define Link**. The web UI remembered the public keys of the actors and providers running, so you just need to pick them out of a dropdown selector at the top of the form, as shown below. For the 'Values' field, you can select a port number for the HTTP Server listener, and enter `port=8080` if you want to use port 8080 on localhost. (there should be no spaces before or after the equals sign). If you want the HTTP listener to use a different ip address, use "address" instead of port, and provide both IP address and the port, as in `address=10.1.1.1:8080`.
+The Echo actor and HTTP Server providers are running, but they aren't connected. Since the HTTP server provider hasn't been _linked_ to any actor yet, it hasn't yet open a port to listen to web requests to forward to the actor. To allow the actor and provider to communicate, they need to be linked. We could link them with wash cli (`wash ctl link put ...`), using the Actor ID and the Provider ID from the inventory, but this time we'll link them in the web UI.
 
-⚠️ `Docker` If you're running this example using the docker image, you'll need to use `address=0.0.0.0:8080` instead of the `port` value. This is required due to the way containers expose ports to host networking environments.
+Go back to the web UI and click **Define Link**. The web UI remembered the public keys of the actors and providers running, so you just need to pick them out of a dropdown selector at the top of the form, as shown below. For the 'Values' field, you'll either need to provide a port to listen on or an address including hostname and port. For simplicity and compatibility, input the following into your 'Values' form:

```

Comment:
you'll either need to provide a port to listen on or an address 
-> you'll need to provide either a port to listen on or an address

Reply:
input the following
-> enter the following